cli/waiter/subcommands/kill.py

```diff
@@ -22,4 +31,20 @@ def register(add_parser):
                         dest='is-service-id', action='store_true')
     parser.add_argument('--timeout', '-t', help='timeout (in seconds) for kill to complete',
                         type=check_positive, default=30)
+
+    ping_group = parser.add_mutually_exclusive_group(required=False)
+    ping_group.add_argument('--no-ping', action='store_false', dest='ping_token',
+                            help='skips pinging the token/service; pinging the token is disabled by default')
+    ping_group.add_argument('--ping', action='store_true', dest='ping_token',
+                            help='pings the token after stopping maintenance.')
```

Comment:
this isn't related to maintenance mode; we should change this to something like:
```suggestion
                            help='pings the token after killing services.')
```

Reply:
Updated message to:
```
pings the token/service after killing service(s).
```